api/src/main/java/org/openmrs/module/fhir2/api/FhirLocationService.java

```diff
@@ -28,4 +30,6 @@
 	Collection<Location> findLocationsByPostalCode(@NotNull String postalCode);
 	
 	Collection<Location> findLocationsByState(@NotNull String state);
+	
+	Collection<Location> findLocationsByTags(@NotNull List<TokenParam> tags);
```

Comment:
Could we revise this to just `findLocationsByTag(@NotNull TokenParam tag)`? The correct implementation for a list of tags is quite a bit more complicated.